### PR TITLE
Remove selectors and new newsLetter page file

### PR DESCRIPTION
**Pull request by mihajlo1985**

### CODE REVIEW

File: Framework/pages/newsLetters.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import { Page } from 'playwright'
+import { expect } from "@playwright/test"
```
Comment (velislavaatanasova) [inline Framework/pages/newsLetters.ts:2]: remove it is not used anywhere.

File: Framework/pages/newsLetters.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+const newsletterDialogCloseButtonLocator = 'button[aria-label=\'close\']';
+const newsletterDialogTitleLocator = '//div[@class="overflow-auto max-h-screen w-full"]//h2';
+const newsletterDialogErrorIconLocator = 'div.icon error';
+const newsletterDialogInfoIconLocator = 'div.icon info';
+const newsletterDialogTextLocator = 'p.newsletter-p';
+const newsletterDialogCancelButton = 'button#cancel-btn'
+const newsletterDialogConfirmButton = 'button#ok-btn';
+
+
+class newsLetter {
```
Comment (velislavaatanasova) [inline Framework/pages/newsLetters.ts:16]: rename to newsletterDialog

File: Framework/pages/newsLetters.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+    async checkNewsletterDialogCloseButtonVisibility(): Promise<boolean> {
+
+        return this.commonActions.isThisVisible(this.Page,newsletterDialogCloseButtonLocator)
+  
+    }
+
+    async closeNewsletterDialogByXButton() {
+
+        this.Page.locator(newsletterDialogCloseButtonLocator).click();
```
Comment (velislavaatanasova) [inline Framework/pages/newsLetters.ts:35]: use click function from commonActions

File: Framework/pages/newsLetters.ts [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    }
+
+    async getNewsletterDialogTitleText(): Promise<String> {
+
+        return this.Page.locator(newsletterDialogTitleLocator).textContent();
```
Comment (velislavaatanasova) [inline Framework/pages/newsLetters.ts:40]: use getText from commonActions

File: Framework/pages/newsLetters.ts [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    }
+    async getNewsletterDialogText(): Promise<String> {
+        
+        return this.Page.locator(newsletterDialogTextLocator).textContent();
```
Comment (velislavaatanasova) [inline Framework/pages/newsLetters.ts:44]: use getText from commonActions

File: Framework/pages/newsLetters.ts [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+    }
+    async checkNewsletterDialogInfoIconVisibility(): Promise<boolean> {
+
+        return this.commonActions.isThisVisible(this.Page,newsletterDialogInfoIconLocator)
+       
+    }
+
+    async closeNewsletterDialogByCancelButton() {
+
+        await this.commonActions.clickElement(this.Page, newsletterDialogCancelButton);
```
Comment (velislavaatanasova) [inline Framework/pages/newsLetters.ts:60]: use click function from commonActions

File: Framework/pages/newsLetters.ts [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+    
+    }
+
+    async confirmNewsletterSignUp() {
+        await this.commonActions.clickElement(this.Page, newsletterDialogConfirmButton);
```
Comment (velislavaatanasova) [inline Framework/pages/newsLetters.ts:65]: use click function from commonActions

**Review by velislavaatanasova — CHANGES_REQUESTED** [review]

File: Framework/pages/newsLetters.ts (hunk 6 above)
Comment (mihajlo1985) [reply Framework/pages/newsLetters.ts:60]: it's clickElement? we are using it :)

File: Framework/pages/newsLetters.ts (hunk 7 above)
Comment (mihajlo1985) [reply Framework/pages/newsLetters.ts:65]: it's commonActions clickElement :)

**Review by velislavaatanasova — APPROVED** [review]

**Pull request merged**